circuit_puzzles/programs/announcer_penalize.clsp

```diff
@@ -32,15 +30,16 @@
         )
```

Comment:
replace penalty_factor_per_interval by (- PERCENT_PRECISION penalty_factor_per_interval) so that Statute can remain the penalty amount in bps?

Reply:
Saves on cost if it's not

Reply:
ok. let's reinsert FACTOR in the Statute name then to avoid confusion